sdk/core/azure-core-amqp/src/models/amqp_header.cpp

```diff
@@ -13,6 +13,7 @@
 
 #include <azure_uamqp_c/amqp_definitions_header.h>
 #elif ENABLE_RUST_AMQP
+#include <azure/core/amqp/internal/common/runtime_context.hpp>
```

Comment:
I am not sure if it is effective for what we want, but FWIW, I think we have a guideline to use `""` when including headers from the same library, and also existing code does that.

Reply:
I'm pretty sure we don't have a guideline, ut we have a convention - slightly different.